### PR TITLE
add host as a custom parameter

### PR DESCRIPTION
**Pull request by LucaMele**
Hello, i think your corsproxy is really usefull, but in my case it didn't open the host as lohalhost but with myy ubuntu user. I guess is because of hapi. So I add a parameter to allow me to configure as well the host. Thank you

### CODE REVIEW

**Comment by gr2m** [comment]
Can you update the README as well, please?

**Comment by LucaMele** [comment]
Ok I have updated it.

**Comment by LucaMele** [comment]
I have updated the readme.

**Comment by LucaMele** [comment]
why does the build fail with the readme update?

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "corsproxy",
+  "version": "0.0.2",
```
Comment (gr2m) [inline package.json]: just wondering, why did you add the version here, and why 0.0.2? Note that version is set dynamically by [semantic-release](https://github.com/semantic-release/semantic-release)

**Comment by gr2m** [comment]
don't worry about the build fail, I'll fix that. I think it's because node version is set to `iojs`, my guess is setting it to `4` will fix that, I'll take care of it

**Comment by LucaMele** [comment]
ah yes sorry i did it for a test.. im taking it out.

**Comment by LucaMele** [comment]
ok thank you

**Comment by gr2m** [comment]
released as [1.4.0](https://github.com/gr2m/CORS-Proxy/releases/tag/v1.4.0)

**Comment by gr2m** [comment]
Thanks a ton, Luca :bouquet: :)

**Comment by LucaMele** [comment]
you are very welcome. Thank you as well
